src/components/CreateNote/CreateNote.tsx

```diff
@@ -0,0 +1,73 @@
+import { ChangeEventHandler, MouseEventHandler } from 'react';
+import {
+  Button,
+  TextField,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+} from '@mui/material';
+import EditIcon from '@mui/icons-material/Edit';
+
+interface ICreateNoteProps {
+  isOpen: boolean;
+  handleClickOpen: MouseEventHandler<HTMLButtonElement>;
+  handleClose: MouseEventHandler<HTMLButtonElement>;
+  handleSave: MouseEventHandler<HTMLButtonElement>;
+  onChangeTitle: ChangeEventHandler<HTMLInputElement | HTMLTextAreaElement>;
+  onChangeDescription: ChangeEventHandler<
+    HTMLInputElement | HTMLTextAreaElement
+  >;
+}
+export const CreateNote = ({
+  isOpen,
+  handleClickOpen,
+  handleClose,
+  handleSave,
+  onChangeDescription,
+  onChangeTitle,
+}: ICreateNoteProps) => (
+  <div>
```

Comment:
can be replaced with <></>
it will not add extra nodes to the DOM
https://legacy.reactjs.org/docs/fragments.html